packages/common/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@affinidi-tdk/common",
+  "version": "0.0.1",
```

Comment:
**chore (blocking):** we need to add `project.json` as in auth-provider. https://github.com/affinidi/affinidi-tdk/blob/main/packages/auth-provider/project.json
otherwise nx will not recognise this package

Reply:
Added